src/packages/cli/templates/model-migration.js

```diff
@@ -58,11 +58,11 @@ export default (name: string, attrs: Array<string> | string): string => {
         ${body}
         table.timestamps();
 
-        table.index([
+        table.index(
           'id',
```

Comment:
I don't think we have to create an additional index for `id` as it will be the primary key.